modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppQt5AbstractCodegen.java

```diff
@@ -91,7 +91,7 @@ public CppQt5AbstractCodegen() {
         typeMapping.put("UUID", "QString");
         typeMapping.put("URI", "QString");
         typeMapping.put("file", "QIODevice");
```

Comment:
@etherealjoy Binary and file are essentially the same so I would suggest mapping `file` to `QByteArray` as well.

Reply:
Makes, sense.

Reply:
Done